go.mod

```diff
@@ -81,6 +81,9 @@ require (
 	go.etcd.io/etcd/api/v3 v3.5.0
 	go.etcd.io/etcd/client/pkg/v3 v3.5.0
 	go.etcd.io/etcd/client/v3 v3.5.0
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.20.0
+	go.opentelemetry.io/otel v0.20.0
+	go.opentelemetry.io/otel/exporters/otlp v0.20.0
```

Comment:
Is there some special reason that we don't use v1.4.0?

https://pkg.go.dev/go.opentelemetry.io/otel@v1.4.0?tab=versions

- v1.* is for v1
- v0.20.0 is for v0

Reply:
There are kubernetes dependencies (etcd, containerd IIRC) which use older versions of otel.  We are stuck on those until we can upgrade dependencies.